recipes/ittapi/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package LANGUAGES CXX)
+
+find_package(ittapi REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE ittapi::ittapi)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

Comment:
this should be in the recipe in the validate method 

https://github.com/conan-io/conan-center-index/blob/e7c65968be736eb9e46dd1c488a3b1d783f1b923/recipes/tuplet/all/conanfile.py#L38-L40

Reply:
Thanks. Actually requirement for C++ 11 was not correct. I have removed it from test files.